src/sjcl-custom/sjcl-ecdsa-signdeterministic.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+let sjcl = require('sjcl');
+/* eslint-disable new-cap */
+
+sjcl.ecc.ecdsa.secretKey.prototype.generateK =
+function(hash, hashObject) {
+
+  let curve = this._curve;
```

Comment:
You should use `const` if the reference will never point to something other than what it is declared at initially. `curve` here is never reassigned so can be const.

However, this code wants to be pushed to sjcl at some point, which doesn't use es6, so you'll want to replace all let/const with var, and use whatever `eslint-disable` directives are required to pacify the linter.